README.md

```diff
@@ -447,8 +447,10 @@ Provides volume levels and state of requested mixers.
 Supported platforms: Linux (required tool: amixer), FreeBSD.
 
 - Arguments (per platform):
-  * Linux: takes the ALSA mixer control as an argument, i.e. `"Master"`,
-    optionally append the card ID or other options, i.e. `"PCM -c 0"`
+  * Linux: takes either a single argument containing the ALSA mixer control as
+    an argument, i.e. `"Master"`, or a table passed as command line arguments
+    to [amixer(1)](https://linux.die.net/man/1/amixer),
+    i.e `{"PCM", "-c", "0"}` or `{"Master", "-D", "pulse"}`.
```

Comment:
How about the following: this makes it more flexible.

Reply:
Looks good and the command seems to work fine with the quotes on my end. I like this method better. Thanks.